src/ui/handlers/run-info-ui-handler.ts

```diff
@@ -15,13 +15,11 @@ import { GAME_HEIGHT, GAME_WIDTH, TEXT_SCALE } from "#app/constants/ui";
 import { UiHandler } from "#app/ui/handlers/abstract-ui-handler";
 import { addBBCodeTextObject, addTextObject, getBBCodeFragment } from "#app/ui/text/text-utils";
 import { addWindow } from "#app/ui/ui-theme";
-import {
-  formatLargeNumberFixedDigits,
-  formatMoney,
-  getPlayTimeString,
-  getPokemonLevelText,
-  isNullOrUndefined,
-} from "#app/utils";
+import { isNullOrUndefined } from "#app/utils/common-utils";
+import { getPokemonLevelText } from "#app/utils/string-utils";
+import { formatMoney } from "#app/utils/string-utils";
+import { formatLargeNumberFixedDigits } from "#app/utils/string-utils";
+import { getPlayTimeString } from "#app/utils/string-utils";
```

Comment:
```suggestion
import {
  formatLargeNumberFixedDigits,
  formatMoney,
  getPlayTimeString,
  getPokemonLevelText
} from "#app/utils/string-utils";
```
again, assuming this won't fit in a single line